docs/com1DFAAlgorithm.rst

```diff
@@ -53,24 +53,35 @@ Particles are initialized according to the release raster extracted from the rel
 and the mass per particle determination method (``massPerParticleDeterminationMethod``) specified in the configuration.
 The mass per particle determination method can be chosen between:
 
-  - MPPDIR= mass per particle direct. The ``massPerPart`` value is taken from the configration
+  - MPPDIR= mass per particle direct. The ``massPerPart`` value is taken from the configuration and is the same for all
+  cells.
 
   - MPPDH= mass per particles through release thickness. The ``massPerPart`` value is computed
     using the release thickness per particle ``deltaTh`` value given in the configuration and the area of
     the release mesh cell: :math:`\mbox{massPerPart} = \rho\times \mbox{cellArea} \times\mbox{deltaTh}`.
 
+  - MPPKR= mass per particles through number of particles per kernel radius. There is not ``massPerPart`` since it can
+  vary from one cell to another depending on the release thickness of the cells. The aim of this method is to ensure a
+  constant density of particles within the snow domain (``nPPK`` particles per kernel radius is the target value).
+  This is related to the SPH method that is used for computing the flow thickness gradient. This method requires a
+  sufficient number of particles to approximate properly the flow thickness gradient. It makes the most sense to combine
+  the MPPKR particle initialization method with the `splitOption` 1. In this combination, the particles will be merged
+  or splitted to keep a rather constant density of particles per kernel radius (:ref:`DFAnumerics:Splitting and merging`).
```

Comment:
```suggestion
  - MPPKR= mass per particles through number of particles per kernel radius. There is no ``massPerPart`` since it can
  vary from one cell to another depending on the release thickness of the cells. The aim of this method is to ensure a
  constant density of particles within the snow domain (``nPPK`` particles per kernel radius is the target value).
  This is related to the SPH method used for computing the flow thickness gradient. It requires a
  sufficient number of particles to properly approximate the flow thickness gradient. It makes the most sense to combine
  the MPPKR particle initialization method with the `splitOption` 1. In this combination, the particles will be merged
  or split to keep a constant density of particles per kernel radius (:ref:`DFAnumerics:Splitting and merging`).
```

Reply:
maybe it would be best to force using splitOption one in this case? Or what is the massPerPart set to in that case- the average over all?

Reply:
not necessary. It could be convenient to use the MPPKR in association with the particle redistribution for example.